daras_ai_v2/asr.py

```diff
@@ -727,21 +742,32 @@ def run_asr(
 
     selected_model = AsrModels[selected_model]
     output_format = AsrOutputFormat[output_format]
+    audio_wav_data = None  # use this if the model expects audio in bytes data
+    filename = "yt_audio"
+    content_type = "audio/wav"
+
     if is_yt_url(audio_url):
-        audio_url, size = download_youtube_to_wav_url(audio_url)
+        audio_wav_data = download_youtube_to_wav(audio_url)
     elif is_gdrive_url(furl(audio_url)):
         meta: dict[str, str] = gdrive_metadata(url_to_gdrive_file_id(furl(audio_url)))
+        filename = meta.get("name", "gdrive_audio")
+        content_type = meta.get("mimeType", "audio/wav")
```

Comment:
this looks wrong, the cotnent type of the uploaded audio is always wav at the end